### PR TITLE
#244 : Adding Full Outer Join

### PR DESCRIPTION
**Pull request by JonasAls89**

### CODE REVIEW

File: training/010_architecture_and_concepts/030_Novice.rst [hunk 1]
```diff
@@ -24,7 +24,50 @@ Make namespaced identifiers for foreign keys - make-ni
...
-All data from input ends up in output
+Full outer join is something you will experience in the Sesam terminology as a "merge". A merge, like the full outer join, retains all entries from i.e. two merged data objects. Graphically, a full outer join will look like the following:
+
+.. figure:: ./media/Full_Outer_Join.png
+   :align: center
+   :alt: Figure – Full Outer Join
+
+   Figure – Full Outer Join
+
+A note on the handling of null values. In Sesam null values is not existing. Meaning, as opposed to a full outer join which will populate empty entries in the join between tables with null values, the merge in Sesam will by default never have to do this. To exemplify, look at the below example:
```
Comment (gabriellsesam) [inline training/010_architecture_and_concepts/030_Novice.rst:35]: In Sesam null values **are** not existing

**Review by gabriellsesam — CHANGES_REQUESTED** [review]
Veldig bra, fant bare en skrivefeil

File: training/010_architecture_and_concepts/030_Novice.rst (hunk 1 above)
Comment (JonasAls89) [reply training/010_architecture_and_concepts/030_Novice.rst]: corrected

**Review by gabriellsesam — APPROVED** [review]

File: training/010_architecture_and_concepts/030_Novice.rst [hunk 2]
```diff
@@ -24,7 +24,50 @@ Make namespaced identifiers for foreign keys - make-ni
 Full outer Join - Merge
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-All data from input ends up in output
+Full outer join is something you will experience in the Sesam terminology as a "merge". A merge, like the full outer join, retains all entries from i.e. two merged data objects. Graphically, a full outer join will look like the following:
+
+.. figure:: ./media/Full_Outer_Join.png
```
Comment (mofsesam) [inline training/010_architecture_and_concepts/030_Novice.rst:29]: Maybe figures from existing course could illustrate this better than a venn diagram? Showing example tables and how the merge.

**Review by mofsesam — APPROVED** [review]

**Pull request merged**